generators/client/templates/angular/src/main/webapp/app/home/_home.route.ts

```diff
@@ -1,14 +1,16 @@
-import { Route } from '@angular/router';
+import { Routes } from '@angular/router';
```

Comment:
That's what I've seen in some angular examples and it looked nicer to read but I can change if you prefer the array.